### ISSUE TITLE
script doesn't recognize wsrep_on variable on percona server

### ISSUE THREAD

**Issue by vchepkov** [major/MySQLTuner-perl#184]
It seems wsrep_on variable is not 'global' on percona cluster and script doesn't see it, reporting replication disabled

For references, I have these packages

Percona-XtraDB-Cluster-galera-3-3.14-1.rhel7.x86_64
Percona-XtraDB-Cluster-server-56-5.6.28-25.14.1.el7.x86_64

```
mysql> SHOW GLOBAL VARIABLES like 'wsrep_on'\G
Empty set (0.00 sec)

mysql> SHOW VARIABLES like 'wsrep_on'\G
*************************** 1. row ***************************
Variable_name: wsrep_on
        Value: ON
1 row in set (0.01 sec)
```

**Comment by jmrenouard** [comment]
Hi @vchepkov 

Can you give me your feedback ?

Actually, a new code is merging show variables and show global variables.

This variables should be now present in %myvar and solve this issue.

**Comment by vchepkov** [comment]
It does recognize variable now, thank you.
But results are confusing. Do you want me to open a different issue?

```
[!!] All cluster nodes dre not detected. wsrep_cluster_size != informations in wsrep_cluster_adress
```

dre is probably a typo and there is no suggestion what needs to be done

```
[!!] Galera SST method is xtrabackup.
```
and at the same time:

```
Variables to adjust:
    set up parameter wsrep_sst_method to xtrabackup
```

**Comment by jmrenouard** [comment]
Hi, 

My awnsers :
1° wsrep_cluster_adress split with comma should give wsrep_cluster_size element.

2° during SST if method is not xtrabackup DONOR node can't receive write.
So It 'is better to get xtrabackup used during SST than other method.
mysqldump is slow and rsync is blocking donor in read mode.
Only xtrabackup allowed Read/write method during SST.

**Comment by vchepkov** [comment]
Still not clear, sorry, what's wrong here?

```
[OK] Galera WsREP is enabled.
[OK] Galera Cluster address is defined: gcomm://10.121.215.31,10.121.215.32,10.121.215.33
[OK] There is 3 nodes.
[!!] All cluster nodes dre not detected. wsrep_cluster_size != informations in wsrep_cluster_adress
```

and about xtrabackup
script generates warning that sst is set to xtrabackup and recommends to set it to xtrabackup at the same time

**Comment by jmrenouard** [comment]
First comment :)
wsrep_cluster_adress split with comma 
In your cases, 
gcomm://10.121.215.31,10.121.215.32,10.121.215.33 should be split in 3 elements.

wsrep_cluster_size is the size of the cluster
if wsrep_cluster_size is not equals to number of element in wsrep_cluster_adress mt is raising an error.

**Comment by jmrenouard** [comment]
xtrabackup is  the best solution for SST situation.
DONOR node is not blocking its write request.

For a 3 nodes situation:
1 node is ready
1 node is DONOR read access only
1 node is unavailable (SST request)

So, in SST situation Galera cluster is only working with one node.
With xtrabackup it works with 2 nodes.

**Comment by jmrenouard** [comment]
Hi @vchepkov 

I push some additionnal information for wsrep_cluster_size /wsrep_cluster_address.

**Comment by vchepkov** [comment]
Here is output from the new version:
```
[OK] Galera WsREP is enabled.
[OK] Galera Cluster address is defined: gcomm://10.121.215.31,10.121.215.32,10.121.215.33
[OK] There is 3 nodes in wsrep_cluster_address.
[OK] There is 3 nodes in wsrep_cluster_size.
[!!] All cluster nodes dre not detected. wsrep_cluster_size != informations in wsrep_cluster_adress
```

**Comment by vchepkov** [comment]
as for sst method it's just not parsed properly:

```
mysql> show variables like 'wsrep_sst_method';
+------------------+---------------+
| Variable_name    | Value         |
+------------------+---------------+
| wsrep_sst_method | xtrabackup-v2 |
+------------------+---------------+
1 row in set (0.00 sec)
```

**Comment by jmrenouard** [comment]
Hi @vchepkov 

Can you give your feedback ?

**Comment by vchepkov** [comment]
Now it doesn't show any recommendations about xtrabackup, but still shows a warning
```
[!!] SST Method is based on xtrabackup.
```
and error about node count is also there

**Comment by jmrenouard** [comment]
Hi @vchepkov

Can you give me your feedback ?

**Comment by vchepkov** [comment]
Warning about 'SST method' went away, thank you.

I did a little bit of research about wsrep_cluster_size

First, this statement will always be true:

```
 if ( $nbNodesSize != 3 or $nbNodesSize != 5 ) {
```

I think you meant to use == here

Second, you wanted to print $nbNodesSize in wsrep_cluster_size, not $nbNodes
This value is 4 in my case, which is indeed not equal node count.

But at the same time, it's a valid configuration, because I also have a garbd node running
It doesn't participate in replication, but it does effect 'votes' count

**Comment by jmrenouard** [comment]
can we close this issue ?

**Comment by vchepkov** [comment]
Yes, it was resolved by #200
